aquarius/events/processors.py

```diff
@@ -96,8 +98,8 @@ def make_record(self, data):
         _record["event"] = {
             "txid": self.txid,
             "blockNo": self.block,
-            "from": self.sender_address,
-            "contract": self.contract_address,
+            "from": self.event.address,
```

Comment:
`event.address` is not equivalent to the `"from"` address.  I recommend using `self.sender_address`.